### PR TITLE
Persons for ef95

### PR DESCRIPTION
**Pull request by DariaElagina**
New entries for persons mentioned in letters of the Collection IVEF95, update of existing entries.

### CODE REVIEW

**Comment by PietroLiuzzo** [comment]
Whoever merges this PR, please remember to use SQUASH AND MERGE. Thanks!

File: new/PRS12864Asenafi.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <revisionDesc>
+            <change who="DE" when="2019-12-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አሸናፊ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAšanāfi</persName>
```
Comment (thea-m) [inline new/PRS12864Asenafi.xml]: ʾAššannāfi (here and in the title)

Comment (thea-m) [reply new/PRS12864Asenafi.xml]: also Amharic, we really need to find a solution to this language assignment in named entities...

File: new/PRS12858Sifaraw.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <revisionDesc>
+            <change who="DE" when="2019-12-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሺፈራው</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Šifarāw</persName>
```
Comment (thea-m) [inline new/PRS12858Sifaraw.xml]: Šifarrāw

File: new/PRS12868Bayyana.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            <change who="DE" when="2019-12-17">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">በየነ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Bayyana</persName>
+                    <persName><roleName type="title">ʾAgāfāri</roleName></persName>
```
Comment (thea-m) [inline new/PRS12868Bayyana.xml]: ʾAggāfāri

File: PRS4001-5000/PRS4357Gabrase.xml [hunk 4]
```diff
@@ -51,19 +51,33 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
          <listPerson>
             <person sex="1">
+               <persName xml:lang="gez" xml:id="n1" type="birth">ገብረ፡ ሥላሴ፡ ባርያ፡ ጋብር፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">Gabra Śǝllāse Bāryā Gābǝr</persName>
                <persName>
-                  
-                  Gäbrä Śǝllase Barya Gabǝr
+                 <roleName type="title">Nǝbura ʾǝd</roleName>
                </persName>
-               <occupation> däǧǧ./nǝburä ǝd </occupation>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply PRS4001-5000/PRS4357Gabrase.xml]: The actual encyclopaedia entry does not mention him being daǧǧazmač, however I haven't checked the whole list of literature oh nim.

File: new/PRS12857Tafarra.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            <change who="DE" when="2019-12-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተፈራ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Tafarrā</persName>
+                    <persName><roleName type="title">fitāwrāri</roleName></persName>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply new/PRS12857Tafarra.xml]: done

File: new/PRS12861Gasaggasa.xml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2019-12-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ገሠገሠ</persName>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply new/PRS12861Gasaggasa.xml]: done

File: new/PRS12864Asenafi.xml (hunk 1 above)
Comment (DariaElagina) [reply new/PRS12864Asenafi.xml]: done, u are right, I will pay more attention to that

**Pull request merged**